contrib/gocql/gocql/gocql.go

```diff
@@ -32,7 +32,6 @@ type params struct {
 	config    *queryConfig
 	keyspace  string
 	paginated bool
-	query     string
 }
 
 // WrapQuery wraps a gocql.Query into a traced Query under the given service name.
```

Comment:
Maybe add a note here recommending usage of WithResourceName in cases where the final query + parameters is expected to be very large or where performance is essential?